packages/sign_in_with_apple/lib/src/authorization_credential.dart

```diff
@@ -155,6 +155,15 @@ AuthorizationCredentialPassword parseAuthorizationCredentialPassword(
 AuthorizationCredentialAppleID parseAuthorizationCredentialAppleIDFromDeeplink(
   Uri deeplink,
 ) {
+  if (deeplink.queryParameters.containsKey('error')) {
+    if (deeplink.queryParameters['error'] == 'user_cancelled_authorize') {
```

Comment:
```suggestion
    // In case the user canceled the sign-in, the URL will have an `error` parameter
    // 
    // The only error code that might be returned is `user_cancelled_authorize`, 
    // which indicates that the user clicked `Cancel`
    //
    // https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_js/incorporating_sign_in_with_apple_into_other_platforms
    if (deeplink.queryParameters['error'] == 'user_cancelled_authorize') {
```